apps/settings/locales/settings.en-US.properties

```diff
@@ -1236,18 +1154,102 @@ licensing-2      = All of the source code to this product is available under lic
+hud-settings-header={{hud-settings-button}}
+hud-devtools=Developer Tools
+# LOCALIZATION NOTE (hud-devtools-header): if you leave the original en-US
+# value "{{hud-devtools}}" in the following string, the localization will be
+# automatically picked from the existing string with ID "hud-devtools". If you
+# need to adapt your localizations to the available space in headers or
+# buttons, you can use different strings here.
+hud-devtools-header={{hud-devtools}}
+hud-logging=Log Changes in ADB
+hud-system=Show System HUD
+hud-problems=Problems
+hud-warnings=Warnings
+hud-errors=Errors
+hud-security=Security Issues
+hud-performance=Performance
+hud-reflows=Reflows
+hud-jank=Jank
+jank-threshold=Jank Threshold
+hud-fps=Frames per Second
+hud-ttl=Time to Load
+hud-memory=Memory
+hud-uss=Unique Set Size
+hud-appmemory=App Memory
+hud-jsobjects=JS Objects
+hud-jsstrings=JS Strings
+hud-jsother=JS Other
+hud-dom=DOM
+hud-style=Style
+hud-other=Other
+software=Software
+devSettings=Developer Settings
+debug=Debug
+developer-menu=Developer Menu
+developer-tools=Developer Tools
+graphics=Graphics
+hardware-composer=Hardware Composer
+paint-flashing=Flash Repainted Area
+dumps-layers=Dump Layers Tree
+dumps-layerscope=Dump LayerScope
+apz-overscroll=Overscrolling
+tiling=Tiling
+low-precision=Low Precision Painting
+low-opacity=Low Precision Transparency
+draw-tile-borders=Draw Tile Borders
+draw-layer-borders=Draw Layer Borders
+log-animations=Log Slow Animations
+remote-debugging-usb=Debugging via USB
+devtools-wifi=DevTools via Wi-Fi
+devtools-device-name=DevTools Wi-Fi Device Name
+devtools-qps=Pseudo-Localization
+debugger-disabled=Disabled
+debugger-adb-only=ADB only
+debugger-adb-and-devtools=ADB and DevTools
+wifi-debugging=Wi-Fi Output in ADB
+bluetooth-debugging=Bluetooth Output in ADB
+bluetooth-snoop-debugging=Bluetooth HCI Sniffing Logs to SD Card
+nfc-debugging=NFC Output in ADB
+ril-debugging=RIL Output in ADB
+console-enabled=Console Cnabled
+cards-view-screenshots=Cards View: Screenshots
+geolocation-debugging-enabled=Geolocation Output in ADB
+geolocation-debugging-gps-locations-ignored=Ignore GPS Locations
+use-reviewer-certs=Use Marketplace Reviewer Certs
+shake-to-log=Shake to Save System Log
 launch-ftu=Launch First Time Use
 no-ftu=No first time use app found
-software-home-button=Software home button
-home-gesture=Home gesture
-continuous-transition=Continuous transition
-debug-gaia-enabled=Gaia debug traces
-showScreenReaderSettings=Show screen reader settings
-app-transition=App transition
-app-suspending=App suspending
-copypaste=Copy/paste
-verbose-app-permissions=Verbose app permissions
-window-management=Window management
+software-home-button=Software Home Button
+home-gesture=Home Gesture
+continuous-transition=Continuous Transition
+debug-gaia-enabled=Gaia Debug Traces
+showScreenReaderSettings=Show Screen Reader Settings
+app-transition=App Transition
+app-suspending=App Suspending
+copypaste=Copy / Paste
```

Comment:
I suppose I leave it as `Copy/Paste`, as `and` could imply new translations and a new string ID.